dev/core/src/com/google/gwt/core/ext/soyc/SourceMapRecorder.java

```diff
@@ -85,30 +83,30 @@ private SourceMapRecorder(int permutationId, List<JsSourceMap> fragmentMaps, Str
 
   private List<SyntheticArtifact> createArtifacts()
       throws IOException, SourceMapParseException {
-    Event event = SpeedTracerLogger.start(CompilerEventType.SOURCE_MAP_RECORDER);
-    List<SyntheticArtifact> toReturn = Lists.newArrayList();
-    ByteArrayOutputStream baos = new ByteArrayOutputStream();
-    SourceMapGeneratorV3 generator = new SourceMapGeneratorV3();
-    int fragment = 0;
-    for (JsSourceMap sourceMap : fragmentMaps) {
-      generator.reset();
-
-      if (sourceRoot != null) {
-        generator.setSourceRoot(sourceRoot);
+    try (SimpleEvent ignored = new SimpleEvent("SourceMap Recorder")) {
+      List<SyntheticArtifact> toReturn = Lists.newArrayList();
+      ByteArrayOutputStream baos = new ByteArrayOutputStream();
+      SourceMapGeneratorV3 generator = new SourceMapGeneratorV3();
+      int fragment = 0;
+      for (JsSourceMap sourceMap : fragmentMaps) {
+        generator.reset();
+
+        if (sourceRoot != null) {
+          generator.setSourceRoot(sourceRoot);
+        }
+        addExtensions(generator, fragment);
+        addMappings(new SourceMappingWriter(generator), sourceMap);
+
+        baos.reset();
+        OutputStreamWriter out = new OutputStreamWriter(baos);
+        generator.appendTo(out, "sourceMap" + fragment);
+        out.flush();
+        toReturn.add(new SymbolMapsLinker.SourceMapArtifact(permutationId, fragment,
+            baos.toByteArray(), sourceRoot));
+        fragment++;
       }
-      addExtensions(generator, fragment);
-      addMappings(new SourceMappingWriter(generator), sourceMap);
-
-      baos.reset();
-      OutputStreamWriter out = new OutputStreamWriter(baos);
-      generator.appendTo(out, "sourceMap" + fragment);
-      out.flush();
-      toReturn.add(new SymbolMapsLinker.SourceMapArtifact(permutationId, fragment,
-          baos.toByteArray(), sourceRoot));
-      fragment++;
+      return toReturn;
     }
-    event.end();
```

Comment:
So here now it used to end the event, but now it just commit it. Is this a change in the how SpeedTracer vs JFR handling things. see my comment on SimpleEvent.